main.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"code.cloudfoundry.org/cli/plugin"
+	"context"
+	"flag"
+	"fmt"
+	"github.com/cloudfoundry/go-cfclient/v3/client"
+	"github.com/cloudfoundry/go-cfclient/v3/config"
+	"github.com/cloudfoundry/go-cfclient/v3/resource"
+	"log"
+	"net/url"
+	"os"
+	"strings"
+)
+
+type lookupRoute struct{}
+
+func main() {
+	plugin.Start(new(lookupRoute))
+}
+
+func (l lookupRoute) Run(cliConnection plugin.CliConnection, args []string) {
+	var err error
+	defer func() {
+		if err != nil {
+			fmt.Printf("error: %s\n", err.Error())
+			os.Exit(1)
+		}
+	}()
+
+	log.SetFlags(0)
```

Comment:
This seems to be the only usage of the `log` package, we should probably remove it completely.